circle.yml

```diff
@@ -0,0 +1,3 @@
+test:
```

Comment:
we're using "circleci 2.0", so there's no longer a circle.yml file.  Take a look at .circleci/config.yml for the right place to put this stuff